gitlab/resource_gitlab_project_share_group.go

```diff
@@ -0,0 +1,157 @@
+package gitlab
+
+import (
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabProjectShareGroup() *schema.Resource {
+	acceptedAccessLevels := make([]string, 0, len(accessLevelID))
```

Comment:
len -1 if we exclude owner?

Reply:
Changed it, I now made this a hardcoded list as there were more invalid options in the accessLevelID list. The only options in the hardcoded list now are options which will actually get accepted by the Gitlab API